### DISCUSSION TITLE
Error Tx while waiting for EOB, timed out... 3.384 >= 3.21399. Starting new burst... RF status: O=0, U=1, L=0

### DISCUSSION THREAD

**Discussion by rajugbv** [srsran/srsRAN_4G#1194] [General]
Hi all,

I am writing to seeking your help regarding an issue I am facing while installing srsRan 5G SA. I have successfully installed srsgnb and open5gs and connected them. However, when I try to connect srsue with srsgnb.

I encounter the following problem:
It appears that srsue can connect with the phy layer, but it cannot be registered by open5gs. I have included the log information below:
**srsue**:-
![Screenshot from 2023-07-05 14-24-32](https://github.com/srsran/srsRAN_4G/assets/41448193/8e879b07-0557-4449-ab29-43ce0cfb0f70)
![Screenshot from 2023-07-05 14-24-35](https://github.com/srsran/srsRAN_4G/assets/41448193/243c4ce4-0355-4947-
b1ec-2c02870b9889)
**gnb conf file:-**
![251243654-5416e936-7286-4193-b06b-f23fc992ae2c](https://github.com/srsran/srsRAN_4G/assets/41448193/10de4656-c23c-4ff9-a4eb-dfd8427c3e54)
![251243653-f6a3e35f-2807-4c79-878d-66f77ee8ccdd](https://github.com/srsran/srsRAN_4G/assets/41448193/24c821e6-04d8-4f4e-9233-b443490b2cb0)
![251243649-5f0d648e-1a2e-4e27-b383-eef1e6349ae1](https://github.com/srsran/srsRAN_4G/assets/41448193/3f8c091d-f3f1-40cf-b420-9d2764f292fe)
![251244861-b0d410c8-f213-4d1c-ae3f-f50eb6c6fa6e](https://github.com/srsran/srsRAN_4G/assets/41448193/d41bdb77-811c-45b7-aa19-5b6be24a629a)
![251244858-53627e73-fa51-4bd3-86b0-e19562720da8](https://github.com/srsran/srsRAN_4G/assets/41448193/6df92398-ef32-4bed-a714-e029304926a3)
![251244856-1cd0c9e1-df45-4ce9-954d-1de91c242714](https://github.com/srsran/srsRAN_4G/assets/41448193/af20a764-c26a-4c37-b13d-e309eec2d52c)
c091d-f3f1-40cf-b420-9d2764f292fe)
![Screenshot from 2023-07-05 14-34-01](https://github.com/srsran/srsRAN_4G/assets/41448193/68c02b96-4b27-427f-b483-32c18e759dd1)
![Screenshot from 2023-07-05 14-34-10](https://github.com/srsran/srsRAN_4G/assets/41448193/2fd4455a-ecb4-4371-9546-280e31d405d7)

I sincerely appreciate your time and assistance in advance. Thank you!

**Comment by CrashxZ** [comment]
Any solutions?
